src/cpp/src/lora/adapter.cpp

```diff
@@ -645,23 +645,9 @@ class LoRAReplaceConstantTransformDynamic : public LoRAReplaceConstantTransform
     }
 };
 
-std::shared_ptr<ov::Node> set_scale(const std::shared_ptr<ov::Node>& A,
-                                    const std::shared_ptr<ov::Node>& alpha,
-                                    const ov::element::Type& element_type) {
-    using namespace ov::op;
-
-    // rank = A_shape[0]
-    auto axis_0 = v0::Constant::create(ov::element::i32, {}, {0});
-    auto rank = std::make_shared<v8::Gather>(std::make_shared<v3::ShapeOf>(A),
-                                             v0::Constant::create(ov::element::i32, {1}, {0}),
-                                             axis_0);
-
-    // scaled_alpha = alpha / rank
-    auto lora_rank = std::make_shared<v0::Convert>(rank, element_type);
-    auto scaled_alpha = std::make_shared<v1::Divide>(alpha, lora_rank);
-
-    return scaled_alpha;
-}
+// TODO [CVS-171466]: LoRA should support two separate parameters: alpha (defined in the config or .safetensors), weight (set by the user)
```

Comment:
```suggestion
// TODO [ticket 171466]: LoRA should support two separate parameters: alpha (defined in the config or .safetensors), weight (set by the user)
```
It's not allowed to mention the domain publicly. Although developers tend to ignore that